tests/integration/test_image_filters.py

```diff
@@ -28,14 +36,567 @@ def test_invert_filter_with_rgb_hsv_image(pil_image, expectation_image):
 
 
 def test_invert_filter_with_rgba_image():
-    inverted_img = invert(RGBA.DIAGNOSTIC_SLIDE_THUMB)
+    inverted_img = imf.invert(RGBA.DIAGNOSTIC_SLIDE_THUMB)
     expected_value = load_expectation(
         "pil-images-rgba/diagnostic-slide-thumb-inverted", type_="png"
     )
 
     np.testing.assert_array_almost_equal(
         np.array(inverted_img), np.array(expected_value)
     )
+
     assert (
         np.unique(np.array(ImageChops.difference(inverted_img, expected_value)))[0] == 0
     )
+
+
+def test_invert_filter_with_gs_image():
+    inverted_img = imf.invert(GS.DIAGNOSTIC_SLIDE_THUMB_GS)
```

Comment:
A test should contain :
- fixtures and assignments
- calls
- assertions

so the tests should be in this format:

def test_invert_filter_with_gs_image():
    img = GS.XXXXX
    expected_value = load_expectaion()

    inverted_img = imf.inverted(img)

    assert A
    assert B
```
```